openshift/manage

```diff
@@ -15,25 +44,36 @@ usage () {
 
   Commands:
   ========
-    configureProof <baseUrl> <payLoadPath> [namespace] [secretName] [key]
+    configureProof <baseUrl> <payLoadPath> [namespace] [secretName] [secretKey]
       - Post a proof configuration to a vc-authn-oidc controller.
       - To update, PUT, use the '-u' (update) option; see examples below.
 
         Examples;
           $0 -p default -e dev configureProof https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca ../proof-configurations/accredited-lawyer/dev/accredited-lawyer.json
           $0 -p default -e dev -u configureProof https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca ../proof-configurations/accredited-lawyer/dev/accredited-lawyer.json
 
-    accepttaa <baseUrl> <secretName> [key] [namespace]
+    accepttaa <baseUrl> [secretName] [secretKey] [namespace]
       - Retrieve the current TAA from the ledger, and submit a TAA accept transaction.
 
         Examples;
-          $0 -p a2a -e dev accepttaa https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca
+          $0 -p a2a -e dev accepttaa https://agent-admin-url-dev.apps.silver.devops.gov.bc.ca
+          $0 -p phg -e dev accepttaa https://public-health-gateway-agent-admin-dev.apps.silver.devops.gov.bc.ca agent-public-health-gateway
+
+    registerSovrin <baseUrl> <stagingnet|buildernet> [secretName] [secretKey] [namespace]
+      - Register the agent on the target ledger as an ENDORSER.
+
+        Examples;
+          $0 -p default -e dev registerSovrin https://agent-admin-url-dev.apps.silver.devops.gov.bc.ca stagingnet
+          $0 -p phg -e dev registerSovrin https://public-health-gateway-agent-admin-dev.apps.silver.devops.gov.bc.ca StagingNet agent-public-health-gateway
```

Comment:
Nit.: stagingnet is case-sensitive, isn't it? It should be all lowercase, I believe.

Reply:
It's set `tolower`.